index.d.ts

```diff
@@ -536,6 +589,10 @@ declare namespace Eris {
     (event: "guildUnavailable" | "unavailableGuildCreate", listener: (guild: UnavailableGuild) => void): T;
     (event: "guildUpdate", listener: (guild: Guild, oldGuild: OldGuild) => void): T;
     (event: "hello", listener: (trace: string[], id: number) => void): T;
+    (event: "interactionCreate", listener: (interaction: PingInteraction | CommandInteraction | ComponentInteraction | UnknownInteraction) => void): T;
+    (event: "commandCreate", listener: (interaction: ApplicationCommand) => void): T;
+    (event: "commandUpdate", listener: (interaction: ApplicationCommand) => void): T;
+    (event: "commandDelete", listener: (interaction: ApplicationCommand) => void): T;
```

Comment:
```suggestion
    (event: "commandCreate", listener: (command: ApplicationCommand) => void): T;
    (event: "commandUpdate", listener: (command: ApplicationCommand) => void): T;
    (event: "commandDelete", listener: (command: ApplicationCommand) => void): T;
```

not really important, but the parameter name should reflect what its value is

Reply:
oop I had forgotten to do this, doesn't matter now